scripts/globals/moghouse.lua

```diff
@@ -42,8 +48,21 @@ function moogleTrigger(player,npc)
             end
         end
 
+        local homeNationFame = player:getFameLevel(player:getNation())
+        local GiveMoogleABreak = player:getQuestStatus(OTHER_AREAS_LOG,GIVE_A_MOOGLE_A_BREAK)
+
         if player:getVar("MoghouseExplication") == 1 then
             player:startEvent(30000)
+        -- Start GiveMoogleABreak
+        elseif player:getLocalVar("QuestSeen") == 0 and GiveMoogleABreak == QUEST_AVAILABLE and homeNationFame >= 3 and
+               player:getVar("MogSafe1Time") + 604800 <= os.time() then -- 604800 One week in Unix epoch
```

Comment:
The wait is until the next conquest tally, not just a full week. I _want_ to say there's a function somewhere that returns the next time until tally, but I'm drawing a blank on its name / where it is. How you'd frame the `safeTime` var (and when) would depend on what that function returns (time remaining until tally, or the absolute next time a tally should occur).

For example, if it returns absolute time, you'd want to set `safeTime` to that absolute time on the furniture placed event, and then here check if the current time is more than the time you stored before.

Reply:
is this what you're looking for:
https://github.com/DarkstarProject/darkstar/blob/5b438424754a60893716759c458147a70a17709c/scripts/globals/common.lua#L80
and you're right... it would be hard to do conquest tally without the bindings...

Reply:
Yep, looks like it, thanks Wiggo!